pkg/cmd/tests/tests.go

```diff
@@ -35,7 +35,11 @@ var Suites = ginkgotest.TestSuites{
 		Description: templates.LongDesc(`
 		Tests that ensure an Scylla Operator is working properly.
 		`),
-		LabelFilter:        fmt.Sprintf("!%s", framework.SerialLabelName),
+		LabelFilter: fmt.Sprintf(
+			"!%s && !%s",
+			framework.SerialLabelName,
+			framework.MultiDatacenterLabelName,
```

Comment:
given `.externalSeeds` is part of the API and it can be tested in a single cluster, why is this not in the conformance suite?

Reply:
We have a dedicated test for testing external seeds in single-datacenter infra as part of the conformance suite. This one is just a multi-dataceneter equivalent. I guess I could merge the two and make a test array out of them, but I'm not sure it's worth it given how much the actual manifests would differ